CycleTests/ACTest.py

```diff
@@ -1,19 +1,28 @@
 '''This code is for Direct Expansion in Cooling Mode'''
 
-from Cycle import DXCycleClass 
+from ACHP.Cycle import DXCycleClass 
+from ACHP.Plots import PlotsClass
 
 #Instantiate the cycle class
 Cycle=DXCycleClass()
 
 #--------------------------------------
-# Cycle parameters
+#         Cycle parameters
 #--------------------------------------
 Cycle.Verbosity = 0 #the idea here is to have different levels of debug output 
 Cycle.ImposedVariable = 'Subcooling'
+Cycle.CycleType = 'DX'
 Cycle.DT_sc_target = 7.0
 Cycle.Mode='AC'
 Cycle.Ref='R410A'
 Cycle.Backend='TTSE&HEOS' #Backend for refrigerant properties calculation: 'HEOS','TTSE&HEOS','BICUBIC&HEOS','REFPROP','SRK','PR'
+Cycle.Oil = 'POE32'
+Cycle.shell_pressure = 'low-pressure'
+Cycle.EvapSolver = 'Moving-Boundary' #choose the type of Evaporator solver scheme ('Moving-Boundary' or 'Finite-Element')
+Cycle.EvapType = 'Fin-tube' #if EvapSolver = 'Moving-Boundary', choose the type of evaporator ('Fin-tube' or 'Micro-channel')
```

Comment:
Micro-channel evaporator is not implemenetd here. I suggest we update the comment